app/src/main/java/com/marverenic/music/utils/SlsInteraction.java

```diff
@@ -0,0 +1,53 @@
+package com.marverenic.music.utils;
+
+import android.content.Context;
+import android.content.Intent;
+
+import com.marverenic.music.model.Song;
+
+public class SlsInteraction {
```

Comment:
Nit on the name here: instances of this classes aren't single interactions – they're delegates that send an interaction. Maybe `SlsMessenger` would make this more clear?

It might also be good to have a quick Javadoc for the class that links to https://github.com/tgwizard/sls, just because SLS isn't a commonly used acronym.

Reply:
>  Maybe `SlsMessenger` would make this more clear?

I'd assume that `SlsMessenger` could be used more than once and `SlsInteraction` to  be used not more than once. This class also represents a single interaction. So, I think `SlsInteraction` is more appropriate name for this.